src/rapier2d-wrapper/src/lib.rs

```diff
@@ -591,6 +591,33 @@ impl<'a> PhysicsHooks for PhysicsHooksCollisionFilter<'a> {
 
 		return true;
     }
+
+    fn modify_solver_contacts(&self, context: &mut ContactModificationContext) {
```

Comment:
it's better to remove this if possible since it doesn't do anything, but it's not dramatic either.

Reply:
I was going to add this feature, but instead added this, and didn't want to delete it all together. If it's all the same will leave it as boilerplate and will continue on it soon. It is a callback that will be used for static body constant speed(applied to rigidbodies and character bodies that touch it).